main.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :name, :age, :parent_permission
+
+  def initialize(name, age, parent_permission)
```

Comment:
- [ ] Kindly define you id as  an `attr_reader :id` and set `@id = Random.rand(1..1000)` in your initialized method